content/en/deployment/strategies/overview.md

```diff
@@ -6,4 +6,109 @@ description: >
+the execution of the deployment. In this strategy you will maintain a 'blue' and a 'green' production environment. The 'blue'
+environment will contain your latest known stable version of your application (usually the version you are replacing with
+the new version) while the 'green' environment contains the changes you are currently deploying. 
+
+Once the 'green' environment is deployed traffic continues to flow exclusively to the 'blue' environment allowing you to 
+validate and test the changes in your 'green' environment. This can be done via manual testing, automated testing or any
+other validation method you may use. 
+
+After validating your changes in the 'green' environment traffic will begin being routed to the 'green' environment which
+becomes the active environment. 
+
+Once traffic is being routed to the 'green' environment you can specify conditions that should be met (additional testing,
+automated canary analysis, manual approval, or a set period of time) after which the 'blue' environment will be shut down. 
+
+### Steps available during the Blue-Green Deployment Strategy
+
+During a blue-green deployment you can specify the following steps as conditions that must be met prior to routing traffic
+to the 'green' environment and before shutting down the 'blue' environment: 
+
+Details on each of these steps and their usage can be found in the [blue-green strategy guide]({{<ref "deployment/strategies/blue-green">}})
+
+- pause - Allows you to wait a specified period of time or wait for an authorized user to issue an approval prior to continuing with the deployment
+- analysis - Uses CD-as-a-Service's Automated Canary Analysis feature to analyze the metrics in the 'green' environment and automatically judge if a deployment should continue
+- exposeServices - Allows you to expose the 'green' environment as a dark deployment using a url generated by CD-as-a-Service. This allows you to access the 'green' environment directly for testing without needing to expose it to your users. More information can be found in the [reference documentation for the CD-as-a-Service Preview Service]({{<ref "reference/deployment/config-preview-link">}})
+- runWebhook - Allows you to execute tests and scripts using your existing tooling, paired with `exposeServices` you can leverage this step to execute automated tests against the 'green' environment.  
+
+### Benefits of the Blue-Green Deployment Strategy
+The blue-green deployment strategy gives you the benefits of running tests against a full copy of your production environment 
+prior to any traffic being routed to the new version. This allows for zero-downtime deployments and reduced risk when exposing
+users to new versions of your application. 
+
+In addition, because the blue-green strategy maintains the 'blue' environment until the shutdown conditions are met it 
+gives you the ability to instantly roll back to a known stable version should any issues be detected during the deployment. 
```

Comment:
```suggestion
The blue/green deployment strategy gives you the benefits of running tests against a full copy of your production environment prior to any traffic being routed to the new version. This gives you for zero downtime deployments and reduced risk when exposing users to new versions of your application. 

In addition, because the blue/green strategy maintains the 'blue' environment until the shutdown conditions are met it, blue/green gives you the ability to instantly roll back to a known stable version should you detect any issues during the deployment. 
```
blue/green
active voice